src/Audis.Location/Audis.Location/Audis.Location.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <Company>softaware gmbh</Company>
+    <Product>Audis</Product>
+    <Authors>softaware gmbh</Authors>
+    <Description>An Audis library which contains location data types.</Description>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageIcon>package-icon.png</PackageIcon>
+    <RepositoryUrl>https://github.com/softawaregmbh/audis-public</RepositoryUrl>
+    <RepositoryType>git</RepositoryType>
+    <PackageTags>Audis</PackageTags>
```

Comment:
Wäre schön, wenn wir hier die Nullable Reference Types aufdrehen (`<Nullable>enable</Nullable>`) und die DTOs entsprechend als nullable/not-nullable markieren.